pallets/creditcoin/src/weights.rs

```diff
@@ -36,23 +36,23 @@ impl<T: frame_system::Config> super::WeightInfo for WeightInfo<T> {
 	// Storage: Creditcoin BidOrders (r:0 w:255)
 	// Storage: Creditcoin Offers (r:0 w:255)
 	// Storage: Creditcoin PendingTasks (r:0 w:510)
-	// Storage: Creditcoin AskOrders (r:0 w:31)
+	// Storage: Creditcoin AskOrders (r:0 w:255)
 	fn on_initialize(a: u32, b: u32, o: u32, d: u32, f: u32, u: u32, c: u32, ) -> Weight {
 		(0 as Weight)
-			// Standard Error: 11_185_000
-			.saturating_add((19_757_000 as Weight).saturating_mul(a as Weight))
-			// Standard Error: 11_185_000
-			.saturating_add((25_578_000 as Weight).saturating_mul(b as Weight))
-			// Standard Error: 11_185_000
-			.saturating_add((26_892_000 as Weight).saturating_mul(o as Weight))
-			// Standard Error: 11_185_000
-			.saturating_add((32_262_000 as Weight).saturating_mul(d as Weight))
-			// Standard Error: 11_185_000
-			.saturating_add((36_736_000 as Weight).saturating_mul(f as Weight))
-			// Standard Error: 11_185_000
-			.saturating_add((26_124_000 as Weight).saturating_mul(u as Weight))
-			// Standard Error: 11_185_000
-			.saturating_add((25_710_000 as Weight).saturating_mul(c as Weight))
+			// Standard Error: 1_106_000
+			.saturating_add((1_432_000 as Weight).saturating_mul(a as Weight))
+			// Standard Error: 1_106_000
+			.saturating_add((1_927_000 as Weight).saturating_mul(b as Weight))
+			// Standard Error: 1_106_000
+			.saturating_add((1_931_000 as Weight).saturating_mul(o as Weight))
+			// Standard Error: 1_106_000
+			.saturating_add((13_944_000 as Weight).saturating_mul(d as Weight))
+			// Standard Error: 1_106_000
+			.saturating_add((21_632_000 as Weight).saturating_mul(f as Weight))
+			// Standard Error: 1_106_000
+			.saturating_add((6_605_000 as Weight).saturating_mul(u as Weight))
+			// Standard Error: 1_106_000
+			.saturating_add((8_111_000 as Weight).saturating_mul(c as Weight))
```

Comment:
these weights changed pretty drastically, is that expected?

Reply:
Weight values are meaningless for dev PRs as long as the benchmark is autogenerating files with the new weight functions and recompiles. Only the CI benchmark routine's PR should be checked for sensible weight values.

Reply:
got it.